src/components/dropdownMenu/DropdownMenuItem.tsx

```diff
@@ -22,17 +23,19 @@ const DropdownMenuItem = ({
   const handleClick = useHandleDropdownMenuItemClick(rest.onClick, onSelect);
 
   return (
-    <button
-      {...rest}
-      onClick={handleClick}
-      type="button"
-      className={getDropdownMenuItemClassName(className)}
-      title={rest.title ?? rest["aria-label"]}
-    >
-      <MenuItemContent icon={icon} shortcut={shortcut}>
-        {children}
-      </MenuItemContent>
-    </button>
+    <DropdownMenuPrimitive.Item className="radix-menu-item">
+      <button
```

Comment:
It might not be in scope of this PR but since you are updating this file, it would be better if we can switch to generic Button component instead